internal/engine/compiler/compiler_post1_0_test.go

```diff
@@ -238,6 +245,65 @@ func TestCompiler_compileMemoryCopy(t *testing.T) {
 	}
 }
 
+func requireBenchmarkNoError(b *testing.B, err error) {
+	if err != nil {
+		b.Fatal(err)
+	}
+}
+
+func BenchmarkCompiler_compileMemoryCopy(b *testing.B) {
+	sizes := []uint32{5, 8, 17, 64, 128, 1000, 16000, 64000}
```

Comment:
please reduce the dimensions unless you are noticing something very surprising between the edges.